examples/with-aptos/src/index.ts

```diff
@@ -0,0 +1,140 @@
+import * as dotenv from "dotenv";
+import * as path from "path";
+import { AptosClient, TxnBuilderTypes, BCS, TransactionBuilder } from "aptos";
+import { input } from "@inquirer/prompts";
+import { Turnkey } from "@turnkey/sdk-server";
+import { bytesToHex } from "@noble/hashes/utils";
+
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+async function main() {
+  const organizationId = process.env.ORGANIZATION_ID!;
+  const turnkeyClient = new Turnkey({
+    apiBaseUrl: process.env.BASE_URL!,
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    defaultOrganizationId: organizationId,
+  });
+
+  const client = new AptosClient("https://fullnode.mainnet.aptoslabs.com/v1");
+  const aptosAddress = process.env.APTOS_ADDRESS!;
+  const aptosPublicKeyHex = process.env.APTOS_PUBLIC_KEY!;
+
+  if (!aptosAddress || !aptosPublicKeyHex) {
+    throw new Error(
+      "Please set your APTOS_ADDRESS and APTOS_PUBLIC_KEY in the .env.local file."
+    );
+  }
+
+  console.log(`Using Aptos address: ${aptosAddress}`);
+  const publicKeyBytes = Uint8Array.from(Buffer.from(aptosPublicKeyHex, "hex"));
+
+  // Check if account exists
+  let accountData = await client.getAccount(aptosAddress).catch(() => null);
+  if (!accountData) {
+    console.log(
+      `Your account does not exist. Please fund your address ${aptosAddress} to proceed.`
+    );
+    process.exit(1);
+  }
+
+  // Create and sign a transaction
+  const recipientAddress = await input({
+    message: "Recipient address:",
+    default: "<recipient_aptos_address>",
+  });
+
+  const amount = 100n; // 100 Octas (minimum practical amount)
+
+  console.log(
+    `\nSending ${amount} Octas (${
+      Number(amount) / 1e8
+    } APT) to ${recipientAddress}`
+  );
+
+  // Prepare the transaction payload
+  const token = new TxnBuilderTypes.TypeTagStruct(
+    TxnBuilderTypes.StructTag.fromString("0x1::aptos_coin::AptosCoin")
+  );
+
+  const entryFunctionPayload =
+    new TxnBuilderTypes.TransactionPayloadEntryFunction(
+      TxnBuilderTypes.EntryFunction.natural(
```

Comment:
looking at this again, this is some interesting syntax lol. would be fun to explore the surface area of Aptos transactions at some point